dev-util/packer/packer-1.8.5.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2022 Gentoo Authors
```

Comment:
Please use our automated tools for committing so stuff like this gets updated automatically. 
https://packages.gentoo.org/packages/dev-util/pkgdev
https://wiki.gentoo.org/wiki/Pkgdev